lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -383,6 +389,17 @@ class ReportUIFeatures {
 
   closeToolsDropdown() {
     this.toolsButton.classList.remove('active');
+    this.toolsButton.setAttribute('aria-expanded', 'false');
+  }
+  
+  openToolsDropdown() {
+      this.toolsButton.classList.add('active');
+      this.toolsButton.setAttribute('aria-expanded', 'true');
+      this._document.addEventListener('keydown', this.onKeyDown);
+      // Wait for the first child to be eligible for focus
+      window.setTimeout(() => {
+        this._dom.find(':first-child', this.toolsDropDown).focus();
+      }, 200);
```

Comment:
I really don't like this but couldn't find another way to give the element focus.

Reply:
Turns out this was caused by a CSS transition on the drop down, I fixed in the latest update using the `transitionend` event.